includes/ajax-override.php

```diff
@@ -225,7 +225,9 @@ function tsmlfe_ajax_feedback() {
 					$chg_types = [];
 				}
 
-				array_push($chg_types, $_POST['choose_meeting_type']);
+				if( isset($_POST['choose_meeting_type']) && !empty($_POST['choose_meeting_type']) ) {
```

Comment:
you can just say `!empty` in php that implies the `isset` -- aka you don't get a warning if it's not set with `empty`

Reply:
Hmmm. Didn't know that. Thx...

Reply:
Wasn't able to make !empty() work the way I wanted, so I rolled back to isset() which worked just fine...

Reply:
strange! there should be no difference at all between `isset($_POST['choose_meeting_type']) && !empty($_POST['choose_meeting_type'])` and `!empty($_POST['choose_meeting_type'])`

Reply:
Sorry Josh, I rolled back on this comment. I was trying to use !empty() in place of isset() in the single-meeting file: <?php if( isset($tsml_hide_contact_information) && $tsml_hide_contact_information !== true) { ?>. Isset worked perfectly but I couldn't get the same result with !empty(). Still learning...